python-prediction-algos/generalized_linear_models-binary-classification/algo.json

```diff
@@ -0,0 +1,277 @@
+
```

Comment:
Nitpick: would be good to lint all JSONs. You can use VSCode format action for that.

Reply:
@david-behar

Reply:
[00ad86f](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/00ad86f43ffec4ed4e83d7406a7d1bf2258f5226)